bindings/python/triplestore/README.md

```diff
@@ -0,0 +1,117 @@
+Triplestore
+===========
+> A Python package encapsulating different triplestores using the strategy
+> design pattern.
+
+This package has no dependencies outside the standard library, but the
+triplestore backends have.
+
+The main class is Triplestore, who's `__init__()` method takes the name of the
+backend to encapsulate as first argument.  It's interface is strongly inspired
+by rdflib.Graph, but simplified when possible to make it easy to use.  Some
+important differences:
+- all IRIs are represented by Python strings
+- blank nodes are strings starting with "_:"
+- literals are constructed with Literal()
+
+The module already provides a set of pre-defined namespaces that simplifies
+writing IRIs. For example:
+
+```python
+>>> from triplestore import RDFS, OWL
+>>> RDFS.subClassOf
+'http://www.w3.org/2000/01/rdf-schema#subClassOf'
+```
+
+New namespaces can be created using the Namespace class, but are usually
+added with the bind() method:
+
+```python
+>>> from triplestore import Triplestore
+>>> ts = Triplestore(backend="rdflib")
```

Comment:
```suggestion
>>> ts = Triplestore(name="rdflib")
```

Reply:
Alternatively change argument in the actual code to backend (I like this).

Reply:
done